apps/Gruntfile.js

```diff
@@ -208,7 +208,8 @@ config.lodash = {
 config.sass = {
   all: {
     options: {
-      outputStyle: (MINIFY ? 'compressed' : 'nested')
+      outputStyle: (MINIFY ? 'compressed' : 'nested'),
+      includePaths: ['../shared/css/']
```

Comment:
The new version is currently in the [frontend_pipeline](https://github.com/code-dot-org/code-dot-org/tree/frontend_pipeline) branch at [lib/frontend/sass.js](https://github.com/code-dot-org/code-dot-org/blob/frontend_pipeline/lib/frontend/sass.js), I've added a [commit](https://github.com/code-dot-org/code-dot-org/commit/0da9a7bfd9497d5ffa0d710bef1111a652515998) that matches this change.